lld/test/COFF/Inputs/directives-multiple.asm

```diff
@@ -0,0 +1,47 @@
+; Presently, the LLVM tooling incorrectly coalesces '.drectve' sections. So,
```

Comment:
I would remove this comment - the assumption that the current tool behaviour is incorrect, is not necessarily true.